lib/active_admin/orm/active_record/comments/views/active_admin_comments.rb

```diff
@@ -35,6 +35,9 @@ def build_comment(comment)
                 comment.author ? auto_link(comment.author) : I18n.t('active_admin.comments.author_missing')
               end
               span pretty_format comment.created_at
+              if authorized?(ActiveAdmin::Auth::DESTROY, comment)
+                text_node link_to I18n.t('active_admin.comments.delete'), comments_url(comment.id), method: :delete, data: { confirm: I18n.t('active_admin.comments.delete_confirmation') }
```

Comment:
Line is too long. [187/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.